src/sentry/sentry_metrics/utils.py

```diff
@@ -16,28 +17,28 @@ class MetricIndexNotFound(InvalidParams):
 
 
 def reverse_resolve_tag_value(
-    use_case_id: UseCaseKey, org_id: int, index: Union[int, str, None], weak: bool = False
+    use_case_key: UseCaseKey, org_id: OrgId, index: Union[int, str, None], weak: bool = False
```

Comment:
nit: can we make the argument names `use_case_id` again? or is there a reason for changing it

Reply:
We can, but there're too many files dependent on it, so I reverted it back to original